CHANGES_NEXT_RELEASE

```diff
@@ -8,4 +8,5 @@ Fix: remove unconditional tracing on xmlTreat()/jsonTreat() which was impacting
 Fix: use NOSIGNAL as libcurl option to avoid crashes in that library. (Issue #1016)
 Add: CLI paramter -cprForwardLimit to cap the number of forwarding request for a single client request. (Issue #1016).
 Add: New operation "GET /v2/entities/[id]" (Issue #950)
+Add: New operation "GET /v2/entities/[id]/attrs/[attr]" (Issue #989)
```

Comment:
nit: GET /v2/entities/{entityId}/attrs/{attrName} better to use URL Templates https://tools.ietf.org/html/rfc6570